### ISSUE TITLE
[BUG] Sound

### ISSUE THREAD

**Issue by nekis-uwu** [bloxstraplabs/bloxstrap#4004]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.1

### What problem did you encounter?

I dont know if this is a bug but when i am playing a game with bloxstrap on, it happens something like some kind of 3d sound, with this i mean that i only hear it if im looking at it, if im not looking at whatever is making that sounds, i cannot hear it, which is very bad on games that requiere you to pay attention to sounds.
I tried to find somewhere in the settings to disable it bur i found nothing related to sound, i also searched in the wiki to see if i could find something realted to it but found nothing.

**Comment by whyd0ineedausername** [comment]
does this happen with normal roblox

**Comment by nekis-uwu** [comment]
no it doesn,t, it only happens when i use bloxstrap on it

**Comment by nekis-uwu** [comment]
when i play on normal roblox, i can hear everything coming from anywhere

**Comment by whyd0ineedausername** [comment]
do you have fflags

**Comment by nekis-uwu** [comment]
yes but i saw what things the flag contained, and it doesnt have anything related to sound, its just related to quality of the game and optimization

**Comment by Tagerz** [comment]
It also happens to me sometimes, that you have to use shiftlock and zoom in almost max to hear any sound.

**Comment by bluepilledgreat** [comment]
120% chance that its something in your flags
